### PR TITLE
[HOPSWORKS-1503] Admin API services

### PR DESCRIPTION
**Pull request by tkakantousis**

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ cookbook 'hopsmonitor', github: "logicalclocks/hopsmonitor-chef", branch: "maste
 cookbook 'glassfish', github: "logicalclocks/chef-glassfish", branch: "master"
 cookbook 'hops_airflow', github: "logicalclocks/airflow-chef", branch: "master"
 cookbook 'hive2', github: "logicalclocks/hive-chef", branch: "master"
-
+cookbook 'kube-hops', github: "logicalclocks/kube-hops-chef", branch: "master"
```
Comment (robzor92) [inline Berksfile]: leftover?

File: files/default/sql/ddl/updates/1.3.0.sql [hunk 2]
```diff
@@ -23,3 +23,6 @@ CREATE TABLE IF NOT EXISTS `feature_store_tag` (
       PRIMARY KEY (`id`),
       UNIQUE KEY `name_UNIQUE` (`name`)
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+ALTER TABLE `hopsworks`.`host_services` CHANGE COLUMN `service` `name` varchar(48) COLLATE latin1_general_cs NOT NULL;
```
Comment (robzor92) [inline files/default/sql/ddl/updates/1.3.0.sql]: did you run it manually? also for the undo?

Comment (tkakantousis) [reply files/default/sql/ddl/updates/1.3.0.sql]: yes

File: Berksfile (hunk 1 above)
Comment (tkakantousis) [reply Berksfile]: fixed

**Pull request merged**